services/edge/admin_rbac.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Test infrastructure for Edge
+    openshift.io/display-name: OCP EDGE QE
+  name: edge
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: image-puller
+  namespace: edge
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ocs-image-puller-binding
+  namespace: ocp
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+- kind: ServiceAccount
+  namespace: ocp
```

Comment:
This should be `edge`

Reply:
Updated, also has to be updated in `services/ocs/admin_rbac.yaml` ?